doc/content/verification_and_validation/val-2a.md

```diff
@@ -0,0 +1,103 @@
+
+!table id=val-2a_set_up_values_TMAP4 caption=Values of material properties.
+| Parameter | Description                          | Value                                                       | Units                 |
+| --------- | ------------------------------------ | ----------------------------------------------------------- | --------------------- |
+| $K_{d,l}$ | dissociation coefficient upstream       | 8.959 $\times 10^{18} (1-0.9999 \exp(-6 \times 10^{-5} t))$ | at/m$^2$/s/Pa$^{0.5}$ |
+| $K_{d,r}$ | dissociation coefficient on downstream  | 1.7918$\times 10^{15}$                                      | at/m$^2$/s/Pa$^{0.5}$ |
+| $K_{r,l}$ | recombination coefficient on upstream   | 1$\times 10^{-27} (1-0.9999 \exp(-6 \times 10^{-5} t))$     | m$^4$/at/s            |
+| $K_{r,r}$ | recombination coefficient on downstream | 2$\times 10^{-31}$                                          | m$^4$/at/s            |
+| $P_{l}$   | pressure on downstream               | 0                                                           | Pa                    |
+| $P_{r}$   | pressure on downstream               | 0                                                           | Pa                    |
+| $D$       | deuterium diffusivity in PCA         | 3$\times 10^{-10}$                                          | m$^2$/2               |
+| $d$       | diameter of PCA                      | 0.025                                                       | m                     |
+| $l$       | thickness of PCA                     | 5$\times 10^{-4}$                                           | m                     |
+| $T$       | temperature                          | 703                                                         | K                     |
+
+
+!alert note title=This validation case replicate TMAP4 rather than TMAP7 due to inconsistent experiment results with [!cite](anderl1985tritium).
+TMAP4 [!citep](longhurst1992verification) and TMAP7 [!citep](ambrosek2008verification) both replicate this validation case. However, they use different model parameters and configurations, and the experimental data presented in the experimental data in [!citep](ambrosek2008verification) for TMAP7 do not correspond to the data published in [!cite](anderl1985tritium). We therefore replicate only the data from TMAP4 in this TMAP8 validation case.
+
+!alert note title=The recombination and dissociation coefficient on upstream are simple exponential.
+Both TMAP4 [!citep](longhurst1992verification) and TMAP7 [!citep](ambrosek2008verification) simplify the recombination and dissociation coefficients on upstream as exponential in time rather than an integrated ion influence in experiment. Thus, TMAP8 uses the same exponential on recombination and dissociation coefficients.
```

Comment:
```suggestion
!alert note title=The upstream recombination and dissociation coefficients as time-dependent exponentials.
Both TMAP4 [!citep](longhurst1992verification) and TMAP7 [!citep](ambrosek2008verification) describe the upstream recombination and dissociation coefficients as time-dependent exponentials rather than mechanistically capture the influence of ion irradiation on material performance. TMAP8 uses the same expressions.
```